### DISCUSSION TITLE
HELP!

### DISCUSSION THREAD

**Discussion by dlanaras** [dlanaras/unbeerable-pun#7] [Show and tell]
# HELP PLEASE THIS IS VERY IMORTANT WHY IS THE CODE MISSING????

**Comment by lunananananananana** [comment]
fuk of

**Comment by dlanaras** [comment]
Never mind found the solution

**Comment by lunananananananana** [comment]
i don't care

**Comment by dlanaras** [comment]
Oh yeah for the ones (not) asking the solution is:
# programming it yourself
